src/nevermined/Assets.ts

```diff
@@ -375,7 +375,7 @@ export class Assets extends Instantiable {
         publisher: Account,
         assetRewards: AssetRewards = new AssetRewards(),
         method: string = 'PSK-RSA',
-        cap?: number,
+        cap: number,
```

Comment:
maybe make `0` by default like in the `royalties` parameter

Reply:
Does make sense create an NFT with cap 0?